### ISSUE TITLE
How I call trait method

### ISSUE THREAD

**Issue by luqasz** [luqasz/avrisp#9]
@phaazon
How I can call `close()` 
https://github.com/luqasz/uctool/blob/refactor/src/main.rs#L28

**Comment by hadronized** [comment]
What do you mean? That code compiles, right?

**Comment by luqasz** [comment]
```
error[E0599]: no method named `close` found for struct `IspMode` in the current scope
   --> src/main.rs:28:9
    |
28  |     isp.close()?;
    |         ^^^^^ method not found in `IspMode`
```

**Comment by luqasz** [comment]
Fixed in bcde0599e7eccb3db703ba93b5e200c623513607
